src/flexlogger/automation/_events.py

```diff
@@ -41,7 +41,7 @@ def __enter__(self):
     def __exit__(self, *args):
         self.unregister_from_events()
 
-    def get_registered_events(self) -> [EventType]:
+    def get_registered_events(self):
```

Comment:
I believe the type hint for this list could be `-> list[EventType]`.

Does that work?

Reply:
Good point. Thanks! I initially removed it because this was generating an error when creating the documention.
-> list[EventType] fixed it.